apps/web/src/components/dialogForms/uploadForm.tsx

```
@@ -1,38 +1,141 @@
+                                    />
+                                    <label
+                                        htmlFor="file-upload"
+                                        className="cursor-pointer flex flex-col items-center justify-center"
+                                    >
+                                        <svg
+                                            className="w-12 h-12 text-muted-foreground mb-2"
+                                            fill="none"
+                                            stroke="currentColor"
+                                            viewBox="0 0 24 24"
+                                            xmlns="http://www.w3.org/2000/svg"
+                                        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)